### PR TITLE
fixed error that occured on first page load if user hadn't searched f…

### PR DESCRIPTION
**Pull request by jasonheeps**
Fixed error that occured on first page load if user hadn't searched for anything yet.
The logic is now that if the string searched for hasn't been initialized yet and therefore is nil, we display all organisms

### CODE REVIEW

**Comment by KseniiaNovinskaia** [comment]
The code looks good to me @Kimon-Haars @sabinacuccibar want to review before merging?

**Comment by jasonheeps** [comment]
Ok, I realized I forgot to remove the "Reset Search" Button for this special case. Added this logic now.

File: app/views/organisms/_searchbar.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
       <input type="text" name="search_name" >
       <input type="submit" value="Search for Organisms">
   </form>
-  <% if params[:search_name] != "" %>
+  <% if !["", nil].include?(params[:search_name]) %>
```
Comment (Kimon-Haars) [inline app/views/organisms/_searchbar.html.erb]: I am mainly trying out this inline-review feature. 😁  isn't if ! better written as unless ?

Comment (jasonheeps) [reply app/views/organisms/_searchbar.html.erb]: yes you're right, will change it!

Comment (jasonheeps) [reply app/views/organisms/_searchbar.html.erb]: done

**Comment by jasonheeps** [comment]
Ok, I figure this is enough review for this bugfix :grin: Will merge it now in order to be able to continue without merge conflicts

**Pull request merged**